contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+        // System generated
+        capacity: 12345234
+
+```
+
+**Consider following example for mirrored pool expansion**
+
+
+Current CSPC is following:
+This CSPC corresponds to a mirrored pool on node `gke-cstor-it-default-pool-1`
+and on node `gke-cstor-it-default-pool-2`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-mirrored
+spec:
+  type: mirrored
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        // System generated
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-2
+        // System generated
+        id: pool-1-bd2
+        // System generated
+        capacity: 12345234
+
+  - hostName: gke-cstor-it-default-pool-2
+
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-2-bd-1
+        // System generated
+        id: pool-2-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-2-bd-2
+        // System generated
+        id: pool-2-bd2
+        // System generated
+        capacity: 12345234
+
+```
+
+Expanding mirrored pools -- the spec will look
+following:
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-mirrored
+spec:
+  type: mirrored
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-1-bd-1
+        id: pool-1-bd1
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-2
+        id: pool-1-bd2
+        // System generated
+        capacity: 12345234
+
+    // New block device group added
+    - groupName: group2
+      blockDevices:
+      - blockDeviceName: pool-1-bd-3
+        id: pool-1-bd3
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-1-bd-4
+        id: pool-1-bd4
+        // System generated
+        capacity: 12345234
+
+  - name: gke-cstor-it-default-pool-2
+
+    groups:
+    - groupName: group1
+      blockDevices:
+      - blockDeviceName: pool-2-bd-1
+        // System generated
+        id: pool-2-bd1
+        // System generated
+        capacity: 12345234
+      - blockDeviceName: pool-2-bd-2
+        // System generated
+        id: pool-2-bd2
+        // System generated
+        capacity: 12345234
+
+    // New block device group added
+    - groupName: group2
+      blockDevices:
+      - blockDeviceName: pool-2-bd-3
+        // System generated
+        id: pool-2-bd3
+        // System generated
+        capacity: 12345234
+
+      - blockDeviceName: pool-2-bd-4
+        // System generated
+        id: pool-2-bd4
+        // System generated
+        capacity: 12345234
+
+```
+
+**NOTE:**
+In case of striped pool expansion no new group but block device in the present
```

Comment:
This is a complex statement. I am trying very hard to understand.

Reply:
Suggestions please.
Idea is : Striped pools do not have groups but others have. 
So if you want to expand striped pool, you add block device in the only group that is present. For others you add groups which contains a list of block devices.